### DISCUSSION TITLE
Clearing browser storage

### DISCUSSION THREAD

**Discussion by joshbenandiel** [WalletConnect/walletconnect-monorepo#3532] [Developer Support] [answered]
Just a question is it okay to clear storage when `disconnect` in `useDisconnect` is called? or do you have function to clear the localStorage that is stored by web3modal? Thanks in advance!

**Comment by glitch-txs** [comment, marked as answer]
Storage is not cleared completely but the session is, I don't think is necessary but you can try if it's something you need :)
